packages/scripts/src/admin-proposals/simulateVoteV2.ts

```diff
@@ -0,0 +1,246 @@
+// Description:
+// - Simulate voting affirmatively on any pending Admin Proposals
+
+import { advanceBlockAndSetTime, computeVoteHashAncillary, getRandomSignedInt, getWeb3ByChainId } from "@uma/common";
+import { GovernorV2Ethers, VotingTokenEthers, VotingV2Ethers } from "@uma/contracts-node";
+import { BigNumberish, BytesLike } from "ethers";
+// import { REQUIRED_SIGNER_ADDRESSES, SECONDS_PER_DAY, YES_VOTE } from "../utils/constants";
+import { getContractInstance } from "../utils/contracts";
+
+const SECONDS_PER_DAY = 86400;
+const YES_VOTE = "1";
+const REQUIRED_SIGNER_ADDRESSES = {
+  deployer: "0x2bAaA41d155ad8a4126184950B31F50A1513cE25",
+  foundation: "0x7a3A1c2De64f20EB5e916F40D11B01C441b2A8Dc",
+  account_with_uma: "0xcb287f69707d84cbd56ab2e7a4f32390fa98120b",
+};
+
+// Run:
+// - Check out README.md in this folder for setup instructions and simulating votes between the Propose and Verify
+//   steps.
+// - This script should be run after any Admin proposal UMIP script against a local Mainnet fork. It allows the tester
+//   to simulate what would happen if the proposal were to pass and to verify that contract state changes as expected.
+// - Vote Simulate: NODE_URL_1=http://localhost:9545 node ./packages/scripts/src/admin-proposals/simulateVoteV2.js --network mainnet-fork
+
+const hre = require("hardhat");
+const { toWei } = hre.web3.utils;
+require("dotenv").config();
+const assert = require("assert").strict;
+
+async function simulateVoteV2() {
+  const foundationSigner = await hre.ethers.getSigner(REQUIRED_SIGNER_ADDRESSES["foundation"]);
```

Comment:
it might be cleaner to replace the `REQUIRED_SIGNER_ADDRESSES["foundation"]` with just `FOUNDATION_WALLET` seeing it's used all over the place and you dont use the other wallets imported in the same object.